### PR TITLE
Switch from yarn to npm

### PR DESCRIPTION
**Pull request by mkue**
Due to yarn having performance issues with Docker, we switch back to `npm`.

This is still a draft because I still have these two issues:

- `TypeError: this.libOptions.parse is not a function`
  https://youtrack.jetbrains.com/issue/WEB-57089/ESLint823-TypeError-thislibOptionsparse-is-not-a-function
  
- Some typing problems, no idea why they appeared now...
  ```
  TS2344: Type 'Organisation' does not satisfy the constraint '{ [Key: string]: CMSType; }'.
  ```

### CODE REVIEW

File: .github/workflows/admin-pr.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           firebase setup:emulators:ui
 
       - name: Install Dependencies
-        run: yarn install
+        run: npm install
 
       - name: Run Tests
         run: yarn emulators:test
```
Comment (andrashee) [inline .github/workflows/admin-pr.yml:28]: Shall we completely switch for all commands to npm?

**Review by andrashee — COMMENTED** [review]
Awesome 😎

File: .github/workflows/admin-pr.yml (hunk 1 above)
Comment (mkue) [reply .github/workflows/admin-pr.yml]: Yes, I would switch everywhere. I updated all files now.

**Comment by mkue** [comment]
This is now finally done. The typing issues I encountered were caused by FireCMS and fixed by another version upgrade.

We use now `npm` instead of `yarn` everywhere.

**Review by andrashee — APPROVED** [review]
Yeah very nice 👍. Thanks for looking into this tedious issue

**Pull request merged**